seismic_zfp/conversion_utils.py

```diff
@@ -59,6 +59,11 @@ def make_header_numpy(bits_per_voxel, blockshape, source, header_info, geom):
                          header_info, bits_per_voxel, blockshape, geom)
     # These 4 bytes indicate the data source for the SGZ file. Use 20 to indicate numpy.
     buffer[76:80] = int_to_bytes(20)
+    if use_higher_samples_precision:
+        # higher precision minimum sample time/depth
+        buffer[84:92] = double_to_bytes(source.samples[0])
+        # higher precision sample interval (μs/m)
```

Comment:
What would user expect when higher sample precision is enabled? 
Should we add a comment specifying the data type and precision level?

Reply:
this is defined in the SeismicZFP Header in the file-specification.md